multicluster/Makefile

```diff
@@ -39,7 +39,9 @@ help: ## Display this help.
 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=controller-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	$(KUSTOMIZE) build config/default > build/yamls/antrea-multicluster.yml
+	$(CURDIR)/hack/generate-manifest.sh -g > build/yamls/antrea-multicluster-leader-global.yml
+	$(CURDIR)/hack/generate-manifest.sh -l changeme > build/yamls/antrea-multicluster-leader-namespaced.yml
```

Comment:
Okay got it, I can also just generate a single_clusterset_deployment_leader.yaml which includes both the leader yamls? Thats should be just one line change, please let me know what you think.

Reply:
Today can we run the two MC controllers both in one Namespace? If so, might be good to generate and publish one yaml (but keep the code to generate separate yamls). If not, let us revisit later.

Reply:
Actually thats a good point. I think I can change the code so that both MC controllers are run in the same namespace and we can make it a requirement so that also solves the other problem of having to split the access to resources. Let me think more about this and see if it will work.

Reply:
@luolanzone Do you agree that this would be simpler to do. i.e. to run a cluster as both member and leader:
1. Run Leader MC Controller in kube-system
2. Run Member MC Controller in kube-system
3. User only configure one ClusterSet resource in kube-system
4. Reconcilation will be as follows:
    a. ClusterSet and ClusterClaim in kube-system - by both member and leader MC controller
    b. ServiceExport in any NS - by member MC Controller
    c. ExportResources in kube-system - by leader MC controller
    d. ImportResources in kube-system - by member RemoteCluster's reconciler
Issues would be:
1. When we sync leader clusterset status to member - we should not do that in case cluster is both leader and member.
2. When we have policies that are defined in kube-system and we support policies at leader cluster - maybe we need to say member controller will ignore them even though the reconciler will be invoked?

Compared to the other approach of restricting member to a specific NS and run it in a separate NS from the leader controller. What do you think?